cysimdjson/jsoninter.h

```diff
@@ -155,9 +155,14 @@ inline dom::object to_object(dom::element & value, int * ok) {
 }
 
 
-inline std::string string_view_to_string(std::string_view sv) {
-	//TODO: This creates a string copy
-	return std::string(sv);
+inline PyObject * string_view_to_string(std::string_view sv) {
+	// This creates a string copy, which ensures termination by \0
+	std::string str = std::string(string_view_object);
```

Comment:
... and we can skip `std::string` completely (we don't need to copy string twice).